blocks/core/ff_module/ff_module-form-preview/ff_module-form-preview.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var React = require('react');
+var InlineEdit = require('../ff_module-inline-edit/ff_module-inline-edit');
+
+module.exports = React.createClass({
+	displayName: 'FormPreview',
```

Comment:
Can you add a propTypes declaration? (See ff_module-dropdown-button-component and https://facebook.github.io/react/docs/reusable-components.html)